packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -921,7 +921,7 @@ export default mixins(
 
 .errors {
 	margin-top: var(--spacing-2xs);
-	color: var(--color-danger);
+	color: var(--custom-font-very-light);
```

Comment:
i am confused here. why are you changing error colors? also is this token even real? seems like this token does not actually exist